src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -85,6 +86,7 @@ public function build(ContainerBuilder $container)
     {
         parent::build($container);
 
+        $container->addCompilerPass(new LoggerPass());
```

Comment:
sorry, I meant something like:
`new LoggerPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, -32`
but maybe order doesn't/shouldn't matter in fact?

Reply:
I think it doesn't matter, but I'm not 100% sure. Will add your patch.